src/iframe-helper.js

```diff
@@ -416,8 +416,12 @@ export class SubscriptionApi {
    * @param {boolean} is3p set to true if the iframe is 3p.
    * @param {function(!JsonObject, !Window, string)} requestCallback Callback
    *     invoked whenever a new window subscribes.
```

Comment:
@keithwrightbos was very uncomfortable with adding allow-same-origin (I did it this way originally) for security reasons. We will compare the two risks, and investigate whether there is a third option, such as passing a more specific allowable origin than * into this constructor.